server/bundler/loader.js

```diff
@@ -2,18 +2,17 @@ var config = require( 'config' ),
 	utils = require( './utils' );
 
 function getSectionsModule( sections ) {
-	var dependencies = '',
+	var dependencies = "var controller = require( 'controller' );\n",
 		loadSection = '',
 		sectionLoaders = '';
 
 	if ( config.isEnabled( 'code-splitting' ) ) {
-		dependencies = [
+		dependencies += [
```

Comment:
Super minor, but it'd be nice to change this such that we aren't outputting two separate var statements. Maybe

```js
var dependencies;
if ( config.isEnabled( 'code-splitting' ) ) {
	dependencies = [
		"var page = require( 'page' ),",
		"\tlayoutFocus = require( 'lib/layout-focus' ),",
		"\tReact = require( 'react' ),",
		"\tLoadingError = require( 'layout/error' ),",
		"\tclasses = require( 'component-classes' ),",
		"\tcontroller = require( 'controller' );",
		'\n',
		'var _loadedSections = {};'
	].join( '\n' );
  ...
} else {
	dependencies = "var controller = require( 'controller' );\n";
}
```

We repeat the controller require here, but I think it's preferable. I might be in the minority here.